index.js

```diff
@@ -1,4 +1,5 @@
 var path = require('path');
+require('string.prototype.endswith');
```

Comment:
Hmm, if `endsWith` is only used in one place, I don't see much point in pulling in the dependency. `depExt` already contains the extension, so we should just be able to use that.